### ISSUE TITLE
Nanopore only mode?

### ISSUE THREAD

**Issue by jdeligt** [hsnguyen/assembly#4]
Hi there, this looks like a really cool piece of software and it is doing a lot of the things we have thinking about needing to get to do real-time AMR detection from ONT runs. I was wondering if you had thought about doing a met-flye (or spades) from the first 10000 or so reads (as the seed) and then kick off the streaming mode from there? This would allow you to not need any pre loaded data (I think).

P.S. it would be really nice if the software could run smoothly on GPU enabled ARM devices so that basecalling can run in parallel to the assembly. (this is what we've been thinking about in terms of truly real-time standalone for clinical purposes: https://github.com/sirselim/jetson_nanopore_sequencing)

**Comment by hsnguyen** [comment]
Hi Joep, it's nice to see you here other than AT :) This was one of my project before moving to Queensland Health when I was focusing on bacterial real-time assembly using ONT in 2019. It's on hold since then as I'm the only developer :(
1. npGraph is originally designed for resolving the assembly graph from SPAdes in mind but we did have the same idea of making the tool pre-assemblies free by e.g. using flye, miniasm.. on the first longest reads to have a draft of the assembly graph before the streaming analysis. Unfortunately I didn't have very promising results out of this lower quality graph from a range of test isolate or toy metagenomic data set. The performance of the tool heavily depended on the quality of the input assembly graph. I might not try hard enough though.
2.  I had works on npGraph server (dev) that is using gRPC + protobuf just like the way ReadFish works. I remember that time we had to port our promethION with the powerful GPU as the Guppy server to serve the basecalling purpose. Didn't have the chance to try jetson device but I think it's definitely possible with the current basecalling methods.

I'm willing to continue this ongoing works but just need to find a chance to do so :)
Happy to chat here or Slack.

**Comment by okendojo** [comment]
Could you provide a small "wiki" on how to run this program?
